src/containers/Login/Login.js

```diff
@@ -38,40 +38,42 @@ export class Login extends Component {
       const favoriteIDs = favoritesResponse.data.map(favorite => favorite.movie_id)
       this.props.dispatchAddAllUserFavorites(favoriteIDs)
     } catch (error) {
-      if (error.message === '500') {
-        this.setState({ error: "Email or Password doesn't match" })
-      }
+      this.props.dispatchSetError(error.message)
```

Comment:
Can we pass in the message we want the user to see instead of the error message? It might be a better user experience